zerver/lib/onboarding.py

```diff
@@ -318,22 +318,22 @@ def send_initial_realm_messages(realm: Realm) -> None:
 
     welcome_messages: List[Dict[str, str]] = [
         {
-            "stream": Realm.DEFAULT_NOTIFICATION_STREAM_NAME,
+            "stream": str(Realm.DEFAULT_NOTIFICATION_STREAM_NAME),
```

Comment:
This code might be more happily refactored to just fetch the stream ID once at the start of this function, and then pass that when sending messages, but not a blocker for this PR.

Reply:
We have three different streams, I think we can't do what you suggested... -- or maybe I didn't understand what you said.